public/components/facets/GeocoordinateFacet.vue

```diff
@@ -320,11 +279,11 @@ export default Vue.extend({
             const feature = polygon(
               [
                 [
-                  [xCoord, yCoord],
-                  [xCoord, yCoord + ySize],
-                  [xCoord + xSize, yCoord + ySize],
-                  [xCoord + xSize, yCoord],
-                  [xCoord, yCoord],
+                  [yCoord, xCoord],
```

Comment:
Can you explain in a small comment why you went from [x, y] to [y, x], which is unusual.

Reply:
In leaflet it uses the latlng which is y,x. 
![Screenshot from 2020-10-16 14-59-15](https://user-images.githubusercontent.com/25306965/96298278-3149f500-0fc0-11eb-89bd-53bae62bda74.png)